UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -124,7 +124,7 @@
 				independent organizations.</p>
 
 			<p>This document proposes a shared framework for presenting publication accessibility metadata in a
-				user-friendly manner, so as to offer the information to end users in a way that is easy for them to
+				user-friendly manner,   to offer the information to end users in a way that is easy for them to
```

Comment:
```suggestion
				user-friendly manner &#8212; to offer the information to end users in a way that is easy for them to
```

I'd use an em-dash here as the changes turns this into more of an explanation.